tests/data/test1.zeek.out

```diff
@@ -1,208 +1,5 @@
-##! A test script with all kinds of formatting errors.
-##!
-##! This Zeekygen head comment has multiple lines with more detail
-##! about this module. It spans two lines.
+# A sample file which needs some formatting.
 
-@load foo/bar/baz.zeek # A "preprocessor" line with comment
-@load blum/frub
+# NOTE: More extensive tests should go into unit tests or separate files in `tests/samples`.
```

Comment:
I didn't fully get why we want to keep `tests/data` at all? It's used primarily by `test_recursion.py`, but wonder if that one could instantiate the file on-demand (it already does build the directory tree on-demand).